package.json

```diff
@@ -9,7 +9,7 @@
     "lint:fix": "eslint . --fix  && prettier --write '{src,examples}/**/*.ts' --single-quote --tab-width 4 --print-width 100 --trailing-comma all --parser typescript",
     "build": "tsc src/*.ts src/**/*.ts -d --strictNullChecks --sourceMap --outdir ./lib --lib ES6,ES5",
     "prepublish": "not-in-install && npm run build || in-install",
-    "test": "ava test",
+    "test": "NODE_ENV=test ava test",
```

Comment:
Won't work on windows without doing `cross-env` or something. maybe care

Reply:
I dont - but would if I knew of anyone running these tests on a windows env.